README.md

```diff
@@ -4,18 +4,18 @@
 
 # Mina
 
-Mina is the first cryptocurrency with a lightweight, constant-sized blockchain. This is the main source code repository for the Mina project. It contains code for the OCaml protocol implementation, [website](https://minaprotocol.com), & wallet. Enjoy!
+Mina is the first cryptocurrency with a lightweight, constant-sized blockchain. This is the main source code repository for the Mina project and contains code for the OCaml protocol implementation, the [Mina Protocol website](https://minaprotocol.com), and wallet. Enjoy!
 
 ## Notes
 
-- Mina is still under active development, and APIs are still evolving. If you do build upon the APIs, be aware that breaking changes may occur.
+Mina is still under active development and APIs are evolving. If you build on the APIs, be aware that breaking changes can occur.
```

Comment:
It might be worth mentioning here, there Mina's implementation of the [Rosetta API](https://www.rosetta-api.org/) offers a more stable (and often more useful) inferface for retrieving the blockchain's state. The downside of Rosetta is that it's run as a separate process, which also relies on an archive being connected to a node. The source code for the archive and Rosetta implementation are located in `src/app/archive` and `src/app/rosetta` respectively, although I heard there are plans to move them to separate repositories.